langchain_linkup/search_retriever.py

```diff
@@ -130,6 +148,10 @@ def _get_relevant_documents(
             depth=self.depth,
             output_type="searchResults",
             include_images=False,
```

Comment:
why not making this one a parameter as well? (maybe you have a good reason no to)